bliss.js

```diff
@@ -95,6 +129,19 @@ extend($, {
 		return $.set(document.createElement(tag || "div"), o);
 	},
 
+	hooks: {
+		add: function (name, callback) {
+			this[name] = this[name] || [];
+			this[name].push(callback);
+		},
+
+		run: function (name, env) {
+			(this[name] || []).forEach(function(callback) {
+				callback(env);
+			});
+		}
+	},
+
```

Comment:
i have fixed this merge problem in commit https://github.com/dperrymorrow/bliss/commit/972263edebf7c2e199c8e251c9129836708c2469